stable/rabbitmq/templates/svc.yaml

```diff
@@ -7,9 +7,12 @@ metadata:
     chart: {{ template "rabbitmq.chart" .  }}
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
-{{- if .Values.metrics.enabled }}
   annotations:
-{{ toYaml .Values.metrics.annotations | indent 4 }}
+{{- if .Values.service.annotations }}
```

Comment:
If no annotations are specified for the service or the metrics it will render an empty `annotations:` line in the yaml. 
I would try to avoid that.